src/morris_sensitivity.jl

```diff
@@ -67,8 +67,7 @@ end
 
 function morris_sensitivity(f,p_range,p_steps;relative_scale=false,kwargs...)
     design_matrices = sample_matrices(p_range,p_steps;kwargs...)
-    y1 = f(design_matrices[1][1])
-    effects = [[]]
+    effects = [[] for i=1:length(p)]
```

Comment:
This is not longer type strict. That may not be an issue though.

Reply:
This used to be something like this (https://github.com/JuliaDiffEq/DiffEqSensitivity.jl/commit/5f32fa05999a9f8bcac97a705d7e5f0726489c5b#diff-d2cff749b323379a3ae89b98849b4d6bL71) but this preallocation doesn't allow for fixed parameters